RecoMTD/TrackExtender/plugins/TrackExtenderWithMTD.cc

```diff
@@ -122,21 +181,23 @@ namespace {
     double prob_p;
   };
 
+  enum class TofCalc { cost = 1, segm = 2, mixd = 3 };
```

Comment:
```suggestion
  enum class TofCalc { kCost = 1, kSegm = 2, kMixd = 3 };
```
2.17 in http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1

Reply:
ok